examples/generalized_linear_models/GLM-truncated-censored-regression.ipynb

```diff
@@ -0,0 +1,1356 @@
+{
```

Comment:
<ul><li>Nitpick: You don't really need the second <code>with</code> context</li><li>Set <code>return_inferencedata = True</code> in <code>sample</code> . This will get rid of the ArviZ <code>FutureWarning</code> you see in <code>plot_posterior</code> and, most importantly, it'll make this NB ready for the new PyMC default (it'll return <code>InferenceData</code> starting from v4).</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>Can do. You get samples out of an InferenceData object as in...</p><pre class="ql-syntax" spellcheck="false">fitted_thing.posterior.["m"].values
</pre><p>Is that the kosher way to do it? It works, but just checking as it looks way more clumsy than trace["m"]</p>

Reply:
<p>You can just do <code>fitted_thing.posterior["m"]</code> and the nice thing is that it's still an <code>InferenceData</code> object, which means that you can index by <code>dims</code> and <code>coords</code> instead of plain integer indexing -- very helpful for multi-dim models!</p>

Reply:
<p>done</p>